script/tool/lib/src/version_check_command.dart

```diff
@@ -577,7 +577,9 @@ ${indentation}The first version listed in CHANGELOG.md is $fromChangeLog.
             'repository policy. If this is a false positive, please '
             'add a line starting with\n'
             '$_missingVersionChangeJustificationMarker\n'
-            'to your PR description with an explanation of why it is exempt.');
+            'to your PR description with an explanation of why it is exempt.'
+            'If editing PR description does not work, try rehashing your '
```

Comment:
What does "rehashing" mean? I can maybe guess, but it's not a standard git term that I've ever encountered, so it's not clear that this would clarify for most readers.

I don't think we wast to say "if editing does not work", since that could also happen if, say, someone typos the marker; we should more clearly state that triggerring a re-run won't work.

Reply:
We should also add this to the `!hasChangelogChange` message.